lib/common/xml.c

```diff
@@ -5003,3 +5003,10 @@ crm_element_value(xmlNode * data, const char *name)
     }
     return (const char *)attr->children->content;
 }
+
+void
+crm_destroy_xml(gpointer data)
```

Comment:
Related to that, I would warmly recommend documenting public API functions in the header files (this is what the API user will deal with in the first place).